spec/lib/post_revisor_spec.rb

```diff
@@ -1408,6 +1408,13 @@
               expect(post.reload.revisions.first.hidden).to eq(true)
             end
 
+            it "doesn't increment public_version for hidden revisions" do
+              post_revisor.revise!(admin, raw: post.raw, tags: topic.tags.map(&:name) + ["secret"])
+              post.reload
+              expect(post.version).to eq(2)
+              expect(post.public_version).to eq(1)
+            end
```

Comment:
Do we have a spec that is a combination of "hidden tag change + other changes that will cause a bump in public version"?